api/src/main/java/org/pmiops/workbench/compliance/ComplianceService.java

```diff
@@ -0,0 +1,13 @@
+package org.pmiops.workbench.compliance;
+
+import org.pmiops.workbench.moodle.ApiException;
+import org.pmiops.workbench.moodle.model.BadgeDetails;
+
+import java.util.List;
+
+public interface ComplianceService {
+
+    int getMoodleId(String email) throws ApiException;
```

Comment:
I know we've been lax about this in some places, but it would be great to have some sort of javadoc for public interface methods like this. Even a single-line comment saying what this method expects and returns is better than nothing. For examples, the JiraService and FireCloudService interfaces have reasonable javadocs.